### PR TITLE
Fixes #135

### LINKED ISSUE #135: Show multiple images in city info page
- We get an array of images for a particular city from the server.
- Use that array to show multiple images in city info page
- Right now, we show only 1 image at the top. Change that to create slideshow of the images we have received.

Reference: 
https://github.com/project-travel-mate/Travel-Mate/blob/0f2924b6f91126c5d13d235a1cc0ebac1c06912c/Android/app/src/main/java/io/github/project_travel_mate/destinations/description/FinalCityInfo.java#L252

### PR DESCRIPTION
**Pull request by syedjahangirpeeran**
Only ImageSlider implemented.

### CODE REVIEW

**Comment by Swati4star** [comment]
What was the problem you were facing with the scroll view?

**Comment by Swati4star** [comment]
Post the screenshots also here.

File: Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityImageSliderAdapter.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    @Override
+    public boolean isViewFromObject(@NonNull View view, @NonNull Object object) {
+        return view == object;
+    }
+
+    @NonNull
+    @Override
+    public Object instantiateItem(@NonNull ViewGroup container, int position) {
+        ImageView imageView = new ImageView(mcontext);
+        Picasso.with(mcontext).load(mimagesArray.get(position)).fit().centerCrop().into(imageView);
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityImageSliderAdapter.java]: add .error() also to this

File: Android/app/src/main/java/io/github/project_travel_mate/destinations/description/FinalCityInfoActivity.java [hunk 2]
```diff
@@ -45,8 +49,8 @@
     TextView weatherinfo;
     @BindView(R.id.head)
     TextView title;
-    @BindView(R.id.image)
-    ImageView iv;
+    @BindView(R.id.image_slider)
+    ViewPager iv;
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/destinations/description/FinalCityInfoActivity.java]: change iv to imagesSliderView

**Comment by Swati4star** [comment]
@syedjahangirpeeran Any updates?

**Comment by syedjahangirpeeran** [comment]
screenshot_20180703-004821

File: Android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -107,9 +107,31 @@
         <activity android:name=".destinations.description.FinalCityInfoActivity" />
         <activity android:name=".destinations.description.PlacesOnMapActivity" />
         <activity android:name=".mytrips.AddNewTripActivity" />
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (Swati4star) [inline Android/app/src/main/AndroidManifest.xml]: There are merge conflicts. Remove these.

**Comment by Swati4star** [comment]
@syedjahangirpeeran There are 108 files changes.

Also, don't write fix #issuenumber in every commit. The issue history becomes so messed up. check on the issue page.

File: Android/app/build.gradle.orig [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+apply from: '../config/checkstyle.gradle'
+
+android {
+    compileSdkVersion 27
+    buildToolsVersion "27.0.3"
+    defaultConfig {
+        applicationId "io.github.project_travel_mate"
+        minSdkVersion 21
+        targetSdkVersion 27
+<<<<<<< HEAD
```
Comment (Swati4star) [inline Android/app/build.gradle.orig]: Not yet fixed.

**Comment by Swati4star** [comment]
@syedjahangirpeeran Looks good!

**Pull request merged**